src/AppShed/Extensions/SpreadsheetBundle/Service/GeoService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AppShed\Extensions\SpreadsheetBundle\Service;
+
+use GuzzleHttp\Client;
+
+/**
+ * Description of GeoService
+ *
+ * @author Victor
+ */
+class GeoService {
+
+    public function getGeo($address)
+    {
+        $guzzle = new Client();
+
+        $address = urlencode($address);
+
+        $url = "http://maps.google.com/maps/api/geocode/json?sensor=false&address={$address}";
+
+        $res = $guzzle->get($url);
+
+        if($res->getStatusCode()==200){
+
+            $resp = $res->json();
+
+            if ($resp['status']=='OK') {
+
+                $lati = @$resp['results'][0]['geometry']['location']['lat'];
+                $longi = @$resp['results'][0]['geometry']['location']['lng'];
```

Comment:
Please do not use the `@` - I know its a pain to write `count($resp['results']) && array_key_exists...` - but its much nicer. Look at the API doc and check that you only check the results that could not be there. e.g. there is no need to check that 'results' exists if the doc says it always will!